inference-engine/src/legacy_api/include/legacy/ie_layers.h

```diff
@@ -1025,7 +1025,8 @@ class INFERENCE_ENGINE_INTERNAL_CNNLAYER_CLASS(EltwiseLayer): public CNNLayer {
         Logical_OR,
         Logical_XOR,
         Logical_NOT,
-        Mean
+        Mean,
+        Abs,
```

Comment:
Looks like you don't need to add `Abs` layer to Eltwise enum. Because in `addSpecificCreator` you create basic CNNLayer.

Reply:
guess I need it in parser:
`void FrontEnd::parseAbs(const Model &model, const ie::CNNLayerPtr &layer, const DataVector &inputs, const DataVector &outputs) const {`
    `LayerParams params = {layer->name, "Eltwise", layer->precision};`
    `auto res = std::make_shared<InferenceEngine::EltwiseLayer>(params);`
    `res->_operation = InferenceEngine::EltwiseLayer::Abs;`

Reply:
Looks like after these changes we can get two behavior:

- Convert ngraph to ICNNNetwork creates CNNLayer for `Abs` layer
- Myriad frontend creates EltwiseLayer.

Reply:
I removed changes in `addSpecificCreator`.
So it is now processed in Myriad frontend.

Reply:
My concern was that I don't want to have two behavior.

Looks like after your changes all plugins which work with legacy representation will work with `Abs` layer as with `CNNLayer` and only `VPU` will interpret it as `EltwiseLayer`. Or am I wrong?
Could you align `VPU` behavior?

Reply:
You are right.
But the try to align VPU behavior to the other plugins leaded to failures in gna plugin. And processing Abs layer a bit differently on VPU solved the issue. I think this is the minimum amount of code to support the operation, which does not require changes in other plugins.
But if it's critical, I'll take another look at what I can do about it.
So, is it critical?

Reply:
It is not critical because it is a legacy part and some plugins already migrated to new API.
But it is strange that changes in VPU plugin affect GNA 😄

Reply:
Okay, then I'll leave this part as it is.
Do you have any other comments?